src/docker-outside-of-docker/install.sh

```diff
@@ -196,14 +196,14 @@ else
         apt-get -y install --no-install-recommends moby-cli${cli_version_suffix} moby-buildx
         apt-get -y install --no-install-recommends moby-compose || echo "(*) Package moby-compose (Docker Compose v2) not available for OS ${ID} ${VERSION_CODENAME} (${architecture}). Skipping."
     else
-        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix}
+        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix} docker-buildx-plugin docker-compose-plugin
     fi
 fi
 
 # Install Docker Compose if not already installed  and is on a supported architecture
 if type docker-compose > /dev/null 2>&1; then
     echo "Docker Compose already installed."
-else
+elif [ "${DOCKER_DASH_COMPOSE_VERSION}" = "v1" ]; then
```

Comment:
> Improves:
> 
> No longer needed to install docker-compose v1 if user hasn't asked for it dockerDashComposeVersion: v2. This speeds up the installation of this feature.
> No longer installs compose-switch if user wants docker-compose v1. It's pointless to install, and again, this speeds up the feature.

Looks like we are currently providing forward compatibility to people who want to install `v1`. Your improvements make sense to me, and would improve performance, however I wonder if people are now expecting to use `docker compose` and `docker-compose` with this Feature.

@Chuxel wondering if you have any other thoughts/concerns with this approach.
We'd need to bump the major version of the Feature if we decide to go ahead with the changes.

Reply:
The thing is, the currently already provides `docker compose` with always v2, and `docker-compose` for v1 or v2 depending on how users set `dockerDashCompose`. However, it only works with `moby: true`. I think you guys probably forgot to test `docker compose` or `dockerDashCompose: v2` when using `moby: false`.

Reply:
To summarize: this PR changes nothing from users perspective, it only fixes bugs and optimizes stuff.

Reply:
Yeah, `docker compose` can only be v2. There's a "switch" package you can use to decide whether `docker-compose` is mapped to v1 or v2.  So, I suspect @felipecrs is correct.